test/Amazon.Extensions.CognitoAuthentication.IntegrationTests/CognitoAWSCredentialsTests.cs

```diff
@@ -54,7 +55,11 @@ public async Task TestGetCognitoAWSCredentials()
             AuthFlowResponse context =
                 await user.StartWithSrpAuthAsync(new InitiateSrpAuthRequest()
                 {
-                    Password = password
+                    Password = password,
+                    UserContextData = new UserContextDataType {
```

Comment:
for this test - can we either

1. add an assertion that the new user context data was successfully populated 
2. create a different test to assert that?

Reply:
I am not sure how to check for that.

Reply:
should be fine as it is since this is an integration test